Makefile

```diff
@@ -575,6 +576,7 @@ flexlink: flexdll/Makefile
 	$(MAKE) -C stdlib COMPILER=../boot/ocamlc stdlib.cma std_exit.cmo
 	cd stdlib && cp stdlib.cma std_exit.cmo *.cmi ../boot
 	$(MAKE) -C flexdll MSVC_DETECT=0 TOOLCHAIN=$(TOOLCHAIN) \
+	  OCAML_CONFIG_FILE=../config/Makefile \
 	  TOOLPREF=$(TOOLPREF) CHAINS=$(FLEXDLL_CHAIN) NATDYNLINK=false \
```

Comment:
`TOOLPREF` no longer needs to be passed, as flexdll/Makefile should get it via ../config/Makefile